components/base/fake/fake.go

```diff
@@ -35,11 +35,11 @@ type Base struct {
 	TurningRadius            float64
 	WheelCircumferenceMeters float64
 	Geometry                 []spatialmath.Geometry
-	logger                   golog.Logger
+	logger                   logging.Logger
 }
 
 // NewBase instantiates a new base of the fake model type.
-func NewBase(_ context.Context, _ resource.Dependencies, conf resource.Config, logger golog.Logger) (base.Base, error) {
+func NewBase(_ context.Context, _ resource.Dependencies, conf resource.Config, logger logging.Logger) (base.Base, error) {
```

Comment:
Assume you saw this already but the motion benchmarking [calls this method directly](https://github.com/viamrobotics/motion-testing/blob/main/base_scenes.go#L63) with a `golog.Logger`.

Reply:
Yeah I checked with the MP team. They said it's normal to break that temporarily.